test-tools/issuer-front-end/src/Main.tsx

```diff
@@ -218,6 +283,14 @@ export default function Main(props: WalletConnectionProps) {
                 url: target.value,
             },
         });
+        extractFromSchema(target.value)
```

Comment:
I removed this logic, so that the credential schema is now only fetched from the smart contract and then displayed. I feel that is less confusing if the attribute fields can only change when a new smart contract index is inserted and not because of two different actions (a new smart contract index or a new credential scheme URL input in step 1).

Reply:
I see. That makes sense since the contract must be set anyhow for the registration to work.

Will remove and add to changelog.